lib/java-coverage.gradle

```diff
@@ -14,6 +18,10 @@ configure(projectsWithFlags('java')) {
 
     project.addFlags('coverage')
 
+    jacoco {
+        toolVersion = jacocoVersion
```

Comment:
Can we just use `toolVersion` instead of introducing a new variable?

Reply:
The `jacocoVersion` is also referenced in the `dependencies` block below.

I think the new variable is more appropriate because `jacocoAnt "org.jacoco:org.jacoco.ant:${somewhatToolVersion}"` looks strange.

Reply:
That makes sense. 😄